### PR TITLE
return an additional bool in EnsureCerts

### PR DESCRIPTION
**Pull request by mengqiy**

### CODE REVIEW

**Comment by mengqiy** [comment]
cc @anfernee

File: pkg/admission/cert/writer/certwriter.go [hunk 1]
```diff
@@ -45,7 +45,7 @@ const (
 // CertWriter provides method to handle webhooks.
 type CertWriter interface {
 	// EnsureCert ensures that the webhooks have proper certificates.
-	EnsureCerts(runtime.Object) error
+	EnsureCerts(runtime.Object) (bool, error)
```
Comment (Liujingfang1) [inline pkg/admission/cert/writer/certwriter.go]: Maybe add some comments here for the return values

File: pkg/admission/cert/writer/certwriter.go [hunk 2]
```diff
@@ -114,10 +116,10 @@ func handleCommon(webhook *admissionregistrationv1beta1.Webhook, ch certReadWrit
 	if !bytes.Contains(caBundle, caCert) {
 		webhook.ClientConfig.CABundle = append(caBundle, caCert...)
 	}
-	return nil
+	return changed, nil
 }
 
-func createIfNotExists(webhookName string, ch certReadWriter) (*generator.Artifacts, error) {
+func createIfNotExists(webhookName string, ch certReadWriter) (*generator.Artifacts, bool, error) {
```
Comment (Liujingfang1) [inline pkg/admission/cert/writer/certwriter.go]: What does this `bool` indicate? Seems to be inconsistent with `changed`.

File: pkg/admission/cert/writer/certwriter.go [hunk 3]
```diff
@@ -127,16 +129,16 @@ func createIfNotExists(webhookName string, ch certReadWriter) (*generator.Artifa
 		// This may happen if there is another racer.
 		case isAlreadyExists(err):
 			certs, err = ch.read(webhookName)
-			if err != nil {
-				return certs, err
-			}
+			return certs, true, err
```
Comment (Liujingfang1) [inline pkg/admission/cert/writer/certwriter.go]: Should this be `return certs, false, err`?

**Pull request closed without merging**